ios/ExpoHyperTrack.podspec

```diff
@@ -0,0 +1,34 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'ExpoHyperTrack'
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.platform       = :ios, '12.0'
+  s.swift_version  = '5.4'
+  s.source         = { git: 'https://github.com/hypertrack/sdk-expo' }
+  s.static_framework = true
+
+  s.dependency 'ExpoModulesCore'
+
+  # Swift/Objective-C compatibility
+  s.pod_target_xcconfig = {
+    'DEFINES_MODULE' => 'YES',
+    'SWIFT_COMPILATION_MODE' => 'wholemodule',
+    'OTHER_LDFLAGS' => '-framework "HyperTrack"',
+    'FRAMEWORK_SEARCH_PATHS' => '"$(PODS_ROOT)/HyperTrack/objc/HyperTrack.xcframework/**"'
```

Comment:
@tulushev I had to use this workaround because if I declare `s.dependency = 'HyperTrack'` it'll complain about `target has frameworks with conflicting names`. IMO it's fine because we assume that `hypertrack-sdk-react-native` is installed, which will pull our SDK dependency.